playbooks/dataiku.yml

```diff
@@ -1,40 +1,38 @@
 - hosts: 127.0.0.1
   connection: local
-  become_user: ubuntu
   gather_facts: no
-  roles:
-    - python3.6-base
   vars_files:
     - ./group_vars/dataiku
 
   tasks:
-    - name: I am confused b
-      command: 'whoami'
-      register: myidentity
-
-    - name: my secret identity
-      debug:
-        msg: '{{ myidentity.stdout }}'
-
-    - name: install default-jdk
-      apt:
-        name: openjdk-11-jdk
+    - name: Increase system limits as required by DSS
+      become: true
+      pam_limits:
+        domain: "{{dss_service_user}}"
+        limit_item: "{{item}}"
+        limit_type: "-"
```

Comment:
Using "-" tries to force both soft and hard limits: if possible, I'd recommend choosing one of them so that the behaviour is more predictable.